packages/bpk-component-graphic-promo/src/BpkGraphicPromo.module.css

```diff
@@ -0,0 +1,18 @@
+/*
+ * Backpack - Skyscanner's Design System
+ * 
+ * Copyright 2016 Skyscanner Ltd
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+@keyframes bpk-keyframe-spin{100%{transform:rotate(1turn)}}.bpk-graphic-promo{display:flex}
```

Comment:
Do we have an animation here?

Reply:
I double-checked. CSS files are committed on other packages too. I would expect them to be ignored since they are auto-generated... 🤔

@olliecurtis What is the preference here?

Reply:
I'm less concerned about the inclusion of this file, and more wondering about the `bpk-keyframe-spin`

Reply:
Is it because this uses BpkCard which has animation functionality?

Reply:
Thanks both for the input! 😄

Does the card spin? 😅
I am completely unable to trigger this, if so. I also checked`packages/bpk-component-card/src/BpkCard.module.scss` and `bpk-mixins/src/mixins/_cards.scss` and I can't find any animations there either.

I formatted the CSS file for BPK card locally and checked where this keyframe is used. The answer is nowhere. So, I am assuming this was somehow brought over by the Card component, but not sure what exactly is wrong.

<img width="1103" alt="image" src="https://user-images.githubusercontent.com/65911077/158879574-9604605e-41eb-4d40-946f-fc65b4b0dc57.png">

It looks like an issue with the Card component though and not the Graphic Promo... 😅

Reply:
That was me being silly, I was running `npm run build` on the top level directory. There is no `build` script in `bpk-component-graphic-promo`, so I am not sure how to generate the CSS files again. 😅

I am not sure if there is a similar story for the Card component, i.e. the CSS wasn't updated when the keyframe was removed from the SCSS file. Under any circumstances, is Storybook using CSS files that are stored elsewhere? I am just asking because Storybook is working fine. 🙂

Reply:
No need to worry about the CSS files, they are there from some experimental work we have done so you can safely ignore this and is not used when running with the standard component as it's not shipped in the component😄

Reply:
👍🏼